cmd/restic/main.go

```diff
@@ -71,6 +71,17 @@ directories in an encrypted repository stored on different backends.
 	},
 }
 
+// Distinguish commands that need the password from those that work without
+// (#2951).
```

Comment:
I don't think we need a reference to the original issue here.

Reply:
Replaced by something more of an explanation.